src/components/slice.ts

```diff
@@ -0,0 +1,135 @@
+  "meetingSeries/loadSeries",
+  async () => {
+    const series = await getSeries();
+    return series;
+  }
+);
+
+// New way to deal with slices. Way less code and more simple to understand
+// the state injected can be mutated as you want and it will work like magic
+const meetingSeriesSlice = createSlice({
+  // prefix for our actions
+  name: "meetingSeries",
+  initialState,
+  reducers: {
+    setSeries(state, { payload: series }: PayloadAction<MeetingSeries>) {
+      state.series = series;
+    },
+    updateMeeting(state, action: PayloadAction<Meeting>) {
+      const meetings = state.series!.meetings!.map((meeting) =>
+        meeting.id === meeting.id ? action.payload : meeting
+      );
+
+      state.series!.meetings = meetings;
+    },
+    updateTalkingPointGeneralData(
+      state,
+      { payload }: PayloadAction<TalkingPoint>
+    ) {
+      const { id, linkedTickets, tickets, ...update } = payload;
+      const talkingPoint = selectAgendaItem(state, id);
+      Object.assign(talkingPoint, update);
+    },
+    linkTicket(
+      state,
+      action: PayloadAction<{
+        talkingPointId: string;
+        ticket: TalkingPointTicket;
+      }>
+    ) {
+      const { ticket, talkingPointId } = action.payload;
+      const talkingPoint = selectAgendaItem(
+        state,
+        talkingPointId
+      ) as TalkingPoint;
+      talkingPoint.tickets = [...talkingPoint.tickets, ticket];
+      talkingPoint.linkedTickets = [...talkingPoint.linkedTickets, ticket.id];
+    },
+    unlinkTicket(
+      state,
+      action: PayloadAction<{
+        talkingPointId: string;
+        ticketId: string;
+      }>
+    ) {
+      const { ticketId, talkingPointId } = action.payload;
+      const talkingPoint = selectAgendaItem(
+        state,
+        talkingPointId
+      ) as TalkingPoint;
+      talkingPoint.tickets = talkingPoint.tickets.filter(
+        (t) => t.id !== ticketId
+      );
+      talkingPoint.linkedTickets = talkingPoint.linkedTickets.filter(
+        (id) => id !== ticketId
+      );
+    },
+    moveTalkingPoint(
+      state,
+      action: PayloadAction<{ id: string; newPosition: number }>
+    ) {
+      const { id, newPosition } = action.payload;
+      const currentMeeting = selectCurrentMeeting(state);
+      const talkingPoint = selectAgendaItem(state, id) as TalkingPoint;
+      const agendaItems = currentMeeting!.agendaItems.filter(
+        (item) => id !== item.id
+      );
+
+      agendaItems.splice(newPosition, 0, talkingPoint); // yeah, I know. Just ignore this.
+
+      currentMeeting!.agendaItems = agendaItems;
+    },
+  },
+  extraReducers(builder) {
+    // Updating status based on the state of the request
+    builder
+      .addCase(loadSeries.pending, (state) => {
+        state.status = "loading";
+      })
+      .addCase(loadSeries.fulfilled, (state, action) => {
+        state.status = "success";
+        state.series = action.payload;
+      })
```

Comment:
In this case, there is not really a necessity because we only fetch this once, but in cases like tickets update or something like this we could for example store all the "commands" in an array and then derive the state from it, if there is something there with the state pending then the derived state should be pending, if all succeded then success and if one fails then failed.

As mentioned in a comment async error management is another topic, so if we want to dig deeper we can have something specific for it